hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/TestHoodieIndexDefinition.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.hudi.index;
+
+import org.apache.hudi.common.model.HoodieIndexDefinition;
+
+import org.junit.jupiter.api.Test;
+
+public class TestHoodieIndexDefinition {
+
+  @Test
+  public void testIndexNameWithoutPrefix() {
+    HoodieIndexDefinition indexDefinition = new HoodieIndexDefinition("func_index_")
```

Comment:
What is being tested here? If it was for some local testing purpose, can we remove it?